### DISCUSSION TITLE
Apertura automatica garage

### DISCUSSION THREAD

**Discussion by mkel84** [indomus/forum#3045] [GENERALE - Discussioni generiche]
Salve,
Ho un garage con una serranda elettrica, si apre/chiude con il classico pulsante del sali/scendi. Volevo installare un dispositivo zigbee (il wifi non prende in garage) per l'apertura automatica e un classico sensore che rileva il passaggio per bloccare la serranda in movimento.
Secondo la vostra esperienza cosa posso usare?

Grazie mille.

**Comment by essegi65** [comment]
ci sono diversi dispositivi che puoi usare vanno bene quelli per le tende per esempio, vai sul sito zigbee2mqtt e trovi quelli compatibili
